modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the YAML view
+
```

Comment:
Consider adding an introduction.

Reply:
All modules need an introduction according to https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs.

Reply:
Fixed.